docs/internals/contributing/writing-documentation.txt

```diff
@@ -256,6 +256,25 @@ If a function, attribute, etc. is added, it's also okay to use a
 We can simply remove the ``.. versionadded:: A.B`` annotation without any
 indentation changes when the time comes.
 
+Minimizing images
+-----------------
+
+Optimize image compression where possible.  For PNG files, try OptiPNG and AdvanceCOMP's ``advpng``:
+
+.. code-block:: sh
+
+   $ cd docs/
+   $ optipng -o7 -zm1-9 -i0 -strip all `find . -type f -name "*.png"`
```

Comment:
Which version of OptiPNG do you have?  Older versions raised this, but newer don't.  I have 0.7.5, and would like to add a note about which versions will winge.